schema/resources/assertion.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "arxiv-external-links",
+    "description": "Clearinghouse for relations between arXiv e-prints and external resources",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+      "@type": {
+        "type": "string"
+      },
+      "ArXivID": {
+        "description": "Canonical arXiv e-print identifier",
+        "type": "string"
+      },
+      "RelationID": {
+	"description": "Realtion ID",
+	"type" : "string"
```

Comment:
Looks like we still need a few more properties; see https://github.com/arXiv/arxiv-external-links/blob/414cac349f9cef98dab7b2d0221ddd877a81b6d9/relations/domain.py#L33

Reply:
0f64e45: I think I covered everything.

Reply:
I think it's better to indent it prettier.

Reply:
> I think it's better to indent it prettier.

Done.

Reply:
It's not enough. The code contains many tab characters. You'd better replace them with white spaces first.

Reply:
Moreover, it's better to fix how many spaces per indent; two-character indents or four-character ones.

Reply:
> Moreover, it's better to fix how many spaces per indent; two-character indents or four-character ones.

Will that improve program performance or readability?

Reply:
It's for readability. 
Almost all text editors for programmers can indent with the same length, without tabs. Or you can use a beautifier.

Reply:
> It's for readability.
> Almost all text editors for programmers can indent with the same length, without tabs. Or you can use a beautifier.

Indented properly now, I think.

Reply:
The indentation looks good.
But there are many mismatches between the schema and the classes in `relations/domain.py`...